src/runner_manager_type.py

```diff
@@ -56,7 +55,8 @@ class RunnerManagerClients:
 
 
 @dataclass
-class RunnerManagerConfig:
+# The instance attributes are all required.
```

Comment:
Could any be grouped together into a sub class?

Reply:
I revert to having `CharmState` under the `RunnerManagerConfig`.